browser-client.cpp

```diff
@@ -234,13 +234,13 @@ bool BrowserClient::OnProcessMessageReceived(
 		} else if (name == "getTransitions") {
 			struct obs_frontend_source_list list = {};
 			obs_frontend_get_transitions(&list);
-			std::vector<const char *> transitions_vector;
+			std::vector<Json> transitions_vector;
 			for (size_t i = 0; i < list.sources.num; i++) {
 				obs_source_t *source = list.sources.array[i];
 				transitions_vector.push_back(
 					obs_source_get_name(source));
 			}
-			json = transitions_vector;
+			json = Json::array(transitions_vector);
```

Comment:
nlohmann is smart enough to convert a vector of JSONs to a JSON array of objects. And `Json::array` is json11 only.
```suggestion
			json = transitions_vector;
```

Reply:
Confirmed. Scrapped the change on my side.